core/src/main/java/ai/timefold/solver/core/api/score/stream/ConstraintFactory.java

```diff
@@ -402,6 +415,6 @@ public interface ConstraintFactory {
      */
     @Deprecated(forRemoval = true)
     @NonNull
```

Comment:
Let's review the class for all leftover uses of `@NonNull` now that it's `@NullMarked`.